data_type/data_optimization.ipynb

```diff
@@ -0,0 +1,2756 @@
+      "\n",
+      "Batch number 1, n_rows: 1000\n",
+      "273 occurences of 'forest'\n",
+      "\n",
+      "Batch number 2, n_rows: 1000\n",
+      "515 occurences of 'forest'\n",
+      "\n",
+      "Batch number 3, n_rows: 1000\n",
+      "245 occurences of 'forest'\n",
+      "\n",
+      "Batch number 4, n_rows: 1000\n",
+      "286 occurences of 'forest'\n",
+      "\n",
+      "Batch number 5, n_rows: 1000\n",
+      "421 occurences of 'forest'\n",
+      "\n",
+      "Batch number 6, n_rows: 1000\n",
+      "423 occurences of 'forest'\n",
+      "\n",
+      "Batch number 7, n_rows: 1000\n",
+      "437 occurences of 'forest'\n",
+      "\n",
+      "Batch number 8, n_rows: 1000\n",
+      "184 occurences of 'forest'\n",
+      "\n",
+      "Batch number 9, n_rows: 1000\n",
+      "75 occurences of 'forest'\n",
+      "\n",
+      "Batch number 10, n_rows: 1000\n",
+      "28 occurences of 'forest'\n",
+      "\n",
+      "Batch number 11, n_rows: 1000\n",
+      "11 occurences of 'forest'\n",
+      "\n",
+      "Batch number 12, n_rows: 1000\n",
+      "11 occurences of 'forest'\n",
+      "\n",
+      "Batch number 13, n_rows: 1000\n",
+      "15 occurences of 'forest'\n",
+      "\n",
+      "Batch number 14, n_rows: 1000\n",
+      "26 occurences of 'forest'\n",
+      "\n",
+      "Batch number 15, n_rows: 297\n",
+      "49 occurences of 'forest'\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "import pyarrow.parquet as pq\n",
+    "file_path = \"./sample_data/vector/departement-31/landuse.parquet\"\n",
+    "\n",
+    "parquet_file = pq.ParquetFile(file_path)\n",
+    "\n",
+    "for n, batch in enumerate(parquet_file.iter_batches(batch_size=1000)):  # batch_size: number of rows to read\n",
+    "    print(f\"Batch number {n}, n_rows: {batch.num_rows}\")\n",
+    "    column=batch.column(\"type\").to_pandas()  # select the column 'type' and transform the column to a pandas series\n",
+    "    n_forest = len(column[column == \"forest\"].index)  # count the occurrences of 'forest' \n",
+    "    print(f\"{n_forest} occurences of 'forest'\\n\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6b68bd0c-3d78-4543-ab30-d74873cea7ec",
+   "metadata": {},
+   "source": [
+    "## Datacube\n",
+    "\n",
+    "Datacube formats (and the corresponding libraries) are generally better optimized for the cloud. Let's reuse the `.zarr` file generated in the [data cube formats tutorial](./datacube_formats.ipynb). As explained in that tutorial, `.zarr` files are natively split into chunks to facilitate chunking mechanisms. Another optimization is lazy loading and computing. Lazy loading means that the data is not loaded into memory until needed; only the metadata is stored. This allows the user to define operations that, similarly, are not computed until required. The loading and computing steps are then optimized and executed at the end, improving ressource usage. Here is an example:"
```

Comment:
Better otimized for the cloud comparing to what?

Reply:
I meant that datacube formats can either be natively or easily optimized for cloud platforms (respectively zarr and netcdf), whereas rasters for example may require a bit more work to be cloud-optimized. I agree that my explanation could be clearer and less ambiguous.